website/content/api-docs/system/internal-counters.mdx

```diff
@@ -195,7 +195,8 @@ $ curl \
 
 ## Partial Month Client Count
 
-This endpoint returns the number of clients for the current month, as the sum of active entities and non-entity tokens.
+This endpoint returns the number of clients per namespace for the current month, as the sum of clients calculated from the use 
```

Comment:
**Do you mean to say**: as the sum of clients **are** calculated from the use of active entities and non-entity tokens?

Reply:
No, Just wanted to mean something like [this](https://www.vaultproject.io/api-docs/system/internal-counters#:~:text=This%20endpoint%20returns%20the%20number%20of%20clients%20per%20namespace%2C%20as%20the%20sum%20of%20clients%20calculated%20from%20the%20use%20of%20active%20entities%20and%20non%2Dentity%20tokens.)